inc/saitypes.h

```diff
@@ -556,6 +556,63 @@ typedef struct _sai_attribute_t {
     sai_attribute_value_t value;
 } sai_attribute_t;
 
+typedef enum _sai_bulk_op_type_t {
+
+    /* bulk operation stops on the first failed creation. Rest of objects will use SAI_STATUS_NON_EXECUTED
+     * return status value. */
+    SAI_BULK_OP_TYPE_STOP_ON_ERROR,
+
+    /* bulk operation ignores the failures and continues to create other objects */
+    SAI_BULK_OP_TYPE_INGORE_ERROR,
+} sai_bulk_op_type_t;
+
+/**
+ * @brief Bulk objects creation.
+ *
+ * @param[in] switch_id SAI Switch object id
+ * @param[in] object_count Number of objects to create
+ * @param[in] attr_count List of attr_count. Caller passes the number
+ *         of attribute for each object to create.
+ * @param[in] attrs List of attributes for every object.
+ * @param[in] type bulk operation type.
+ *
+ * @param[out] object_id List of object ids returned
+ * @param[out] object_statuses List of status for every object. Caller needs to allocate the buffer.
+ *
+ * @return #SAI_STATUS_SUCCESS on success when all objects are created or #SAI_STATUS_FAILURE when
```

Comment:
I think you mean:
```..... #SAI_STATUS_SUCCESS on success when all objects are created or a failure code (sai_status_t) when any of the objects failes....```. 
The correct status could be used for further debug/analysis. Can we just clarify the comment so that implementation handles it properly.